Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -1168,7 +1376,71 @@ namespace xr
                         inputSource.AimSpace.Pose.Orientation.W = location.pose.orientation.w;
                     }
                 }
-                
+
+                // Get gamepad data
+                {
+                    auto& inputSource = InputSources[idx];
+                    auto controllerInfo = sessionImpl.ControllerInfo;
+
+                    // Get trigger value data
+                    float currentTriggerValue;
+                    bool triggerChangedSinceLastSync = m_impl->QueryControllerFloatAction(actionResources.ControllerGetTriggerValueAction, session, currentTriggerValue);
```

Comment:
uber nit: Could check the return value directly in the `if` instead of assigning the result to a variable (since the out reference will get assigned if the condition evaluates as true), but feel free to ignore this if you'd rather keep it as-is